cmd/frontend/graphqlbackend/search_results.go

```diff
@@ -881,6 +881,14 @@ func (r *searchResolver) Results(ctx context.Context) (*SearchResultsResolver, e
 	case *query.OrdinaryQuery:
 		return r.evaluateLeaf(ctx)
 	case *query.AndOrQuery:
+		//get settings to check if SearchUpperCase is active or not and, if so, run transformer
+		settings, err := decodedViewerFinalSettings(ctx)
+		if err != nil {
+			return nil, err
+		}
+		if *settings.SearchUpperCase {
+			q.Query = query.SearchUpperCase(q.Query)
+		}
```

Comment:
```suggestion
		if v:= settings.SearchUpperCase; v != nil && *v {
			q.Query = query.SearchUpperCase(q.Query)
		}
```

If the setting is not set, we'll get a nil-deref.